src/imagery/i.eodag/i.eodag.html

```diff
@@ -25,15 +25,250 @@ <h3>WARNING: I.EODAG IS UNDER DEVELOPMENT. THIS IS AN EXPERIMENTAL VERSION.</h3>
 through the <b>id</b> option.
 
 <p>
-To be able to download data through <em>i.eodag</em>, the user will need to register for the
+To be able to download data through <em>i.eodag</em>, the user will
+need to register for the
 providers of interest.
-<em>i.eodag</em> reads the user credentials, which the user should have acquired,
-from the EODAG YAML config file.  User have to specify the config file path
-location through the <b>config</b> option, otherwise <em>i.eodag</em>
-will use the credentials found in the default config file <em>~/.config/eodag/eodag.yml</em>
+<em>i.eodag</em> reads the user credentials, which the user should have
+acquired, from the EODAG YAML config file.  User have to specify the config
+file path location through the <b>config</b> option, otherwise <em>i.eodag</em>
+will use the credentials found in the default config file
+<em>~/.config/eodag/eodag.yml</em>
+
+<h2>NOTES</h2>
+<h3>Querying</h3>
+<p>
+Querying, aka. Filtering, is a method introduced to i.eodag to filter search
+results based on an extended list of Products' properties, called
+<a href="https://eodag.readthedocs.io/en/stable/notebooks/api_user_guide/4_search.html?highlight=queryables#Queryables">queryables</a>.<br>
+The <b>print</b> option can be used to get hint of the avaible queryables,
+refere to <a href="#list_queryables_example">examples</a>.<br>
+For example, to get the queryables for the product "S2_MSI_L2A" that is
+offered by Copernicus Data Space Ecosystem, aka. "cop_dataspace":
+<div class="code"><pre>
+i.eodag print=queryables provider=cop_dataspace producttype=S2_MSI_L2A
+</pre>
+</div>
+Note that the <b>print</b> option only gives a subset of the avaible queryables,
+and you can in fact use any of the product's properties, for filtering.
+If you are not sure about the all the available properties,
+you can run a generic search with the <b>j</b> flag and setting <b>limit=1</b>,
```

Comment:
Maybe better to implement that as convenience for the user instead of using the `list_queryables` function that can be missleading in terms of search options...

Reply:
Agree, I will create another PR with that, probably after getting done with i.landsat.download, if it is not urgent.